static/src/js/components/CollectionSelector/CollectionSelector.jsx

```diff
@@ -28,6 +28,7 @@ import { useLazyQuery, useSuspenseQuery } from '@apollo/client'
 
 import { GET_PERMISSION_COLLECTIONS } from '@/js/operations/queries/getPermissionCollections'
 
+import useAppContext from '@/js/hooks/useAppContext'
```

Comment:
When we import files us `@` syntax into files that have imports not using it, we should be taking that opportunity to update the existing imports to use the newer `@` syntax.

Reply:
done